### PR TITLE
Creating new Hardware Configuration chapter (JSC-1633)

### PR DESCRIPTION
**Pull request by asettle**

### CODE REVIEW

**Comment by asettle** [comment]
@ajaeger here is the rendered version

[book-storage-deployment_color_en.pdf](https://github.com/SUSE/doc-ses/files/4360105/book-storage-deployment_color_en.pdf)

**Comment by ajaeger** [comment]
Thanks, LGTM.

**Comment by asettle** [comment]
@tbazant this is the stuff that needs to be forward-ported to SES 7. Have a review and then add it :)

File: xml/deployment_hwrecommend.xml [hunk 1]
```diff
@@ -211,42 +211,231 @@
...
     </para>
-   </listitem>
-   <listitem>
+    <itemizedlist mark="bullet" spacing="normal">
+     <listitem>
+      <para>
+       Separate &adm; with 4 GB RAM, four cores, 1 TB
+       storage capacity. This is typically the &smaster; node.
+       &ceph; services and gateways, such as &mon;, &mds;,
+       Metadata Server, &osd;, &ogw;, or &ganesha; are
```
Comment (tbazant) [inline xml/deployment_hwrecommend.xml]: this is duplicity, MDS is Metadata Server
```suggestion
       &osd;, &ogw;, or &ganesha; are
```

File: xml/deployment_hwrecommend.xml [hunk 2]
```diff
@@ -211,42 +211,231 @@
...
+         must not be used for any other physical, containerized, or
+         virtualized workload.
+       </para>
+     </listitem>
+     <listitem>
+       <para>
+         If any of the gateways (&iscsi;, &rgw;, &ganesha;, &mds;, ...) are deployed
+         within VMs, these VMs must not be hosted on
+         the physical machines serving other cluster roles. (This is
+         unnecessary, as they're supported as collocated services.)
```
Comment (tbazant) [inline xml/deployment_hwrecommend.xml]:
```suggestion
         the physical machines serving other cluster roles. (This is
         unnecessary, as they are supported as collocated services.)
```

File: xml/deployment_hwrecommend.xml [hunk 3]
```diff
@@ -211,42 +211,231 @@
...
+           &mds; nodes (one active/one hot standby): 32 GB RAM, 8 core processor,
+           RAID 1 SSDs for disk.
+         </para>
+         <para>
+           See <xref linkend="sysreq-mds"/> for further recommendation.
+         </para>
+       </listitem>
+       <listitem>
+        <para>
+         One SES management node: 4 GB RAM, 4 core processor, RAID 1 SSDs for
```
Comment (tbazant) [inline xml/deployment_hwrecommend.xml]:
```suggestion
         One SES &adm;: 4 GB RAM, 4 core processor, RAID 1 SSDs for
```

File: xml/deployment_hwrecommend.xml [hunk 4]
```diff
@@ -323,10 +512,16 @@
     </listitem>
     <listitem>
      <para>
-      For disk performance reasons, we recommend using bare metal for OSD nodes
-      and not virtual machines.
+      For disk performance reasons, OSD nodes are bare metal nodes.
+      No other workloads should run on an OSD node unless it is a minimal
+      setup of monitors and managers.
```
Comment (tbazant) [inline xml/deployment_hwrecommend.xml]:
```suggestion
      setup of &mon;s and &mgr;s.
```

File: xml/deployment_hwrecommend.xml [hunk 5]
```diff
@@ -211,42 +211,231 @@
...
+        <para>
+         The total capacity of the cluster should be sized so that even
+         with one node unavailable, the total used capacity (including
+         redundancy) does not exceed 80%.
+        </para>
+       </listitem>
+       <listitem>
+        <para>
+         25 Gb Ethernet or better, bonded for internal cluster and external
+         public network each
```
Comment (tbazant) [inline xml/deployment_hwrecommend.xml]:
```suggestion
         public network each.
```

File: xml/deployment_hwrecommend.xml [hunk 6]
```diff
@@ -211,42 +211,231 @@
...
+        </para>
+       </listitem>
+       <listitem>
+        <para>
+         56+ OSDs per storage cluster
```
Comment (tbazant) [inline xml/deployment_hwrecommend.xml]:
```suggestion
         56+ OSDs per storage cluster.
```

File: xml/deployment_hwrecommend.xml [hunk 7]
```diff
@@ -211,42 +211,231 @@
...
+        </para>
+       </listitem>
+       <listitem>
+        <para>
+         See <xref linkend="sysreq-osd"/> for further recommendation
```
Comment (tbazant) [inline xml/deployment_hwrecommend.xml]:
```suggestion
         See <xref linkend="sysreq-osd"/> for further recommendation.
```

File: xml/deployment_hwrecommend.xml [hunk 8]
```diff
@@ -211,42 +211,231 @@
...
+        </para>
+       </listitem>
+      </itemizedlist>
+     </listitem>
+     <listitem>
+      <para>
+       Dedicated physical infrastructure nodes
```
Comment (tbazant) [inline xml/deployment_hwrecommend.xml]:
```suggestion
       Dedicated physical infrastructure nodes.
```

File: xml/deployment_hwrecommend.xml [hunk 9]
```diff
@@ -323,10 +512,16 @@
...
-      For disk performance reasons, we recommend using bare metal for OSD nodes
-      and not virtual machines.
+      For disk performance reasons, OSD nodes are bare metal nodes.
+      No other workloads should run on an OSD node unless it is a minimal
+      setup of &mon;s and &mgr;s.
      </para>
     </listitem>
+    <listitem>
+      <para>
+        SSDs for Journal with 6:1 ratio SSD journal to OSD
```
Comment (tbazant) [inline xml/deployment_hwrecommend.xml]:
```suggestion
        SSDs for Journal with 6:1 ratio SSD journal to OSD.
```

**Pull request merged**